crates/iota/src/validator_commands.rs

```diff
@@ -538,20 +538,13 @@ impl IotaValidatorCommand {
                     validator_address,
                     print_unsigned_transaction_only,
                 )?;
-                // Make sure the address is a validator
+                // Make sure the address is a committee member
```

Comment:
Is that really so? What does `IotaValidatorCommand::RegisterBridgeCommittee` do?

Reply:
I thought the `BridgeCommittee` was the subset of Committee, but after more investigation, in the Move code (`bridge::committee::register`) we use 
```rust
let validators = system_state.active_validator_addresses();
````
, which indicates that active validators (including `ValidatorSetV2`, see the `iota_system::validator_set::active_validator_addresses`) are allowed to be registered as members of `BridgeCommittee`.

Hence I updated comment, printed info, and the code in [7f267e1](https://github.com/iotaledger/iota/pull/6015/commits/7f267e10bf8d42bf8d98bbb70c3727a580e73250).